### PR TITLE
Replace most of the "link extensions" section with a non-normative note

### PR DESCRIPTION
**Pull request by yoavweiss**
Closes https://github.com/w3c/preload/issues/95

***
[Preview](https://s3.amazonaws.com/pr-preview/yoavweiss/preload/remove_link_extensions.html) | [Diff](https://s3.amazonaws.com/pr-preview/w3c/preload/5d13ba9...yoavweiss:7876c53.html)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -239,13 +238,11 @@ <h2 id="x2.link-type-preload">Link type "<code>preload</code>"</h2>
...
     <section>
-      <h2>`link` element extensions</h2>
-      <p>[[!HTML]] defines the `as` content and IDL attributes. The state of
-      the `as` content attribute is taken into account in the above processing
-      model, which will set the <a>request destination</a> as well as the
-      corresponding <a>request type</a> and <a>request initiator</a>. This is necessary
-      to guarantee correct prioritization, request matching, application of the
-      correct [[!CSP3]] policy, and setting of the appropriate `Accept` request
+      <h2>`as` attribute</h2>
+      <p class=note>[[!HTML]] [defines](https://html.spec.whatwg.org/#attr-link-as) the
```
Comment (igrigorik) [inline index.html:242]: Since this is a note, should the references be informative? [[CSP3]], etc?

**Review by igrigorik — COMMENTED** [review]
LGTM, modulo ref nit :)

File: index.html (hunk 1 above)
Comment (yoavweiss) [reply index.html:242]: changed

File: index.html [hunk 2]
```diff
@@ -239,13 +238,11 @@ <h2 id="x2.link-type-preload">Link type "<code>preload</code>"</h2>
...
-      the `as` content attribute is taken into account in the above processing
-      model, which will set the <a>request destination</a> as well as the
-      corresponding <a>request type</a> and <a>request initiator</a>. This is necessary
-      to guarantee correct prioritization, request matching, application of the
-      correct [[!CSP3]] policy, and setting of the appropriate `Accept` request
+      <h2>`as` attribute</h2>
+      <p class=note>[[HTML]] [defines](https://html.spec.whatwg.org/#attr-link-as) the
+      `as` content and IDL attributes. The attribute is necessary to guarantee
+      correct prioritization, request matching, application of the correct
+      [[!CSP3]] policy, and setting of the appropriate `Accept` request
```
Comment (igrigorik) [inline index.html]: Guessing, other way around.. We already have normative HTML ref, right? If so, keep it as such, but CSP3 should be informative?

Comment (yoavweiss) [reply index.html]: switched

**Review by igrigorik — APPROVED** [review]

**Comment by igrigorik** [comment]
👍

**Pull request merged**